test/build.test.js

```diff
@@ -15,13 +15,23 @@ const fs = require("fs");
 const test = require("ava");
 const Eleventy = require("@11ty/eleventy");
 
-test("Builds minified CSS", async function (t) {
+test.before(async function () {
     let elev = new Eleventy();
     await elev.write();
+});
+
+test("Builds minified CSS from CSS", async function (t) {
     let mainCss = fs.readFileSync("dist/assets/styles/app.css", "utf8");
     let timelineCss = fs.readFileSync("dist/assets/styles/pages/timeline.css", "utf8");
-    let sassCss = fs.readFileSync("dist/assets/styles/sass.css", "utf8");
     t.is(mainCss, "*{box-sizing:border-box}button{font-family:inherit;font-size:1rem}*+*{margin-top:var(--space,1em)}");
     t.is(timelineCss, ".timeline ul{padding-inline-start:0;list-style:none}");
+});
+
+test("Builds minified CSS from Sass", async function (t) {
+    let sassCss = fs.readFileSync("dist/assets/styles/sass.css", "utf8");
     t.is(sassCss, "a{color:#600}mark{background-color:#f90}");
 });
+
+test("Builds minified JavaScript", async function (t) {
+    t.true(fs.existsSync("dist/assets/scripts/app.js"));
```

Comment:
For CSS and Sass tests you check the output of the file, but for JS it looks like you only check the presence of it. Is it possible to also check that the JS file contains the expected content?